packages/react-radio/src/components/Radio/Radio.test.tsx

```diff
@@ -11,14 +10,56 @@ describe('Radio', () => {
     primarySlot: 'input',
   });
 
-  // TODO add more tests here, and create visual regression tests in /apps/vr-tests
+  it('renders a default state', () => {
+    render(<Radio />);
+    expect(screen.getByRole('radio')).toBeTruthy();
```

Comment:
Do you need to use `screen`? So far most of our unit tests destructure the `getBy*` functions out of the render function like: 

```
const { getByRole } = render(<Radio />);
```

Not a blocker but I feel we should align on usage style.

Reply:
Yeah I agree we should align on style. I went with `screen` because that's what is recommended by the testing-library documentation: https://testing-library.com/docs/queries/about/#using-queries

That said, I don't have a strong opinion one way or the other. I'd be happy to change it to align with other tests.

Reply:
The way I read the docs is that they're recommending querying `screen` rather than `document`, not recommending using it in all cases. I agree that `const { getByRole }` seems more idiomatic in general, if the rendered element is within the container.

Reply:
Sounds good! I updated the tests to no longer use `screen`.